great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -1278,8 +1279,8 @@ def get_inspector(self) -> sqlalchemy.engine.reflection.Inspector:
 
         return self._inspector  # type: ignore[return-value]
 
-    @contextmanager  # type: ignore[arg-type]
-    def get_connection(self) -> sqlalchemy.Connection:  # type: ignore[misc]
+    @contextmanager
+    def get_connection(self) -> Generator[sqlalchemy.Connection, None, None]:
         """Get a connection for executing queries.
```

Comment:
Explanation

https://adamj.eu/tech/2021/07/04/python-type-hints-how-to-type-a-context-manager/